### ISSUE TITLE
Email sub not working

### ISSUE THREAD

**Issue by ranstigate** [ranstigate/ranstigate-blog#1]

**Comment by ranstigate** [comment]
I was testing from incognito and it still errors out, cache is not the problem.

![image](https://user-images.githubusercontent.com/56231190/66285217-438f4e80-e8e9-11e9-9748-8f5fa4c674c8.png)

**Comment by ranstigate** [comment]
I created a new post to test email notification but there was none. . 


![image](https://user-images.githubusercontent.com/56231190/66285301-ae408a00-e8e9-11e9-8b42-40b24fad1b8b.png)

**Comment by deysanjay74** [comment]
it's working

**Comment by ranstigate** [comment]
> it's working

Yes this is fine, I will just keep trying few more times and close this.

**Comment by ranstigate** [comment]
HI 

I checked this on mobile in a completely new browser window and it gives this error. 

![image](https://user-images.githubusercontent.com/56231190/68071331-900e6280-fd9f-11e9-93d5-b17054a320f0.png)


This is not a cache issue, its something else completely. 

Also the email id typed inside is not visible to the user, please change the color of the id typed inside to black, 

![image](https://user-images.githubusercontent.com/56231190/68071371-f1363600-fd9f-11e9-89a7-fdd29704fedf.png)

**Comment by ranstigate** [comment]
Hi 

This is a second try with a different address and again got the same error. .

you can try this by using id from here https://www.tempmailaddress.com . .it allows you to create a fake id with a mail box to test this feature.

![image](https://user-images.githubusercontent.com/56231190/68071463-9b15c280-fda0-11e9-86bf-7cbe18d788cf.png)
